scripts/eurostat/health_determinants/fruits_vegetables/process.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file.
+"""
+import os
+import sys
+import pandas as pd
+
+_COMMON_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(1, _COMMON_PATH)
+# pylint: disable=wrong-import-position
+from common.euro_stat import EuroStat
+# from common.euro_stat import sv_list
+# pylint: enable=wrong-import-position
+
+
+class EuroStatFruitsVegetables(EuroStat):
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+    _import_name = "fruits_vegetables"
+
+    _mcf_template = ("Node: dcid:{sv}"
+                     "\n{sv_name}"
+                     "\ntypeOf: dcs:StatisticalVariable"
+                     "\npopulationType: dcs:Person"
+                     "{denominator}"
+                     "{education}"
+                     "{residence}"
+                     "{n_portion}"
+                     "{gender}"
+                     "{countryofbirth}"
+                     "{citizenship}"
+                     "{lev_limit}"
+                     "{coicop}"
+                     "{frequenc_fruitsvegetables}"
```

Comment:
nit: this should be `frequency_fruitsvegetables` (note the missing `y`).

Reply:
The spelling is according to the source input file.